.circleci/config.yml

```diff
@@ -53,6 +53,50 @@ jobs:
           command: |
             python -m twine upload dist/*
 
+deploy_docker:
+    machine:
+      image: ubuntu-1604:202007-01
```

Comment:
okay, sorry, I'm realizing that mriqc might not be a good reference here. this image is deprecated and will stop working next week :(

Reply:
yes - that seems to be the easiest fix https://circleci.com/blog/ubuntu-14-16-image-deprecation/